vendor/processors/base.py

```diff
@@ -222,16 +293,23 @@ def process_payment(self):
         This is where the core of the payment processing happens.
         """
         # Gateway Transaction goes here...
+        self.create_payment_model()
+
         self.transaction_submitted = True
-        pass
+
+        self.save_payment_transaction_result(True, 1, {'gateway_response': 'any outpout'})
```

Comment:
is this the correct text?  Should 'any output' be 'no output'?  Is there a dynamic value that is returned?

Reply:
how about this:
`self.save_payment_transaction_result(True, 1, {'gateway_response': 'output dependent on payment gateway'})`
?